src/python/grpcio_testing/grpc_testing/__init__.py

```diff
@@ -117,3 +384,19 @@ def strict_fake_time(now):
     """
     from grpc_testing import _time
     return _time.StrictFakeTime(now)
+
+
+def channel(service_descriptors, time):
```

Comment:
My thoughts are to punt on it for now and maybe come back to it later. It's nonbreaking-to-users to do this now, find out it was a mistake, and add a default later, but the reverse is not the case.